html/semantics/forms/attributes-common-to-form-controls/resources/dirname.js

```diff
@@ -0,0 +1,11 @@
+function onIframeLoaded(cb, selector="iframe") {
```

Comment:
This function should take _t_ as parameter so the `load` listener below can be wrapped in `t.step_func_done`. And then `cb` won't have to be wrapped cause it runs synchronously as part of that.

Reply:
That would not work in `dirname-only-if-applies.html`, as I'm not using an `async_test` there. I wanted to have separate subtests for `input` and `textarea` and from my understanding, I cannot have two `async_test`s at the same time.

Reply:
I see, but that is a bit messy mixing both an explicit_done and having several nested tests.

That makes it very easy to create a harness failure which would result in the additional tests not getting registered.

Reply:
Okay, so I applied the change and now use two asynchronous tests.
I believe, it's better then to have done in the function name. Also, using `step` only on `cb` leaves the function less nested.